src/Http/RequestFactory.php

```diff
@@ -218,4 +231,38 @@ public function createHttpRequest()
 		return new Request($url, NULL, $post, $files, $cookies, $headers, $method, $remoteAddr, $remoteHost, $rawBodyCallback);
 	}
 
+	/**
+	 * @param bool $usingTrustedProxy
```

Comment:
> Where did you get these information?

By looking at Nette code.


> I was following the coding standards.

Nope. Citation „The @param annotation is followed by two spaces, type and optionally by space and description.“

> Omitting parameter name breaks PhpStorm and is not recommended by phpDocumentator

This is well known but **consistency is a lot more important**. I even wrote Nette RFC to change that but there is unresolved issue with alignment.